ReportingServicesTools/Functions/CatalogItems/Get-RsSubscription.ps1

```diff
@@ -0,0 +1,141 @@
+        
+        .DESCRIPTION
+            This script retrieves information about subscriptions for a report.
+        
+        .PARAMETER Path
+            Specify the path to the report.
+        
+        .PARAMETER ReportServerUri
+            Specify the Report Server URL to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Credential
+            Specify the password to use when connecting to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Proxy
+            Report server proxy to use.
+            Use "New-RsWebServiceProxy" to generate a proxy object for reuse.
+            Useful when repeatedly having to connect to multiple different Report Server.
+        
+        .EXAMPLE
+            Get-RsSubscription -Path '/path/to/my/report'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver using current user's credentials and retrieve details of subscriptions found at '/path/to/my/report'.
+        
+        .EXAMPLE
+            Get-RsSubscription -ReportServerUri 'http://remote-machine:8080/reportserver_sql16' -Path '/path/to/my/report'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://remote-machine:8080/reportserver_sql16 using current user's credentials and retrieve details of subscriptions found at '/path/to/my/report'.
+    #>
+
+    [cmdletbinding()]
+    param
+    (
+        [Alias('ItemPath')]
+        [Parameter(Mandatory = $True, ValueFromPipeline = $true)]
+        [string[]]
+        $Path,
+        
+        [string]
+        $ReportServerUri,
+        
+        [Alias('ReportServerCredentials')]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+        
+        $Proxy
+    )
+    
+    Begin
+    {
+        $Proxy = New-RsWebServiceProxyHelper -BoundParameters $PSBoundParameters
+    }
+    Process
+    {
+        foreach ($Item in $Path)
+        {
+            try
+            {
+                Write-Verbose "Retrieving subscriptions contents..."
+                #$Proxy.ListSubscriptions($Item)
+                
+                $subscriptions = $Proxy.ListSubscriptions($Item)
+                Write-Verbose "Subscriptions retrieved successfully!"
+
+                $extSettings = $null
+                $desc = $null
+                $active = $null
+                $status = $null
+                $eventType = $null
+                $matchData = $null
+                $values = $null
+                
+                Write-Verbose "GetSubscriptionProperties"
+                foreach ($subscription in $subscriptions) {
```

Comment:
our code convention for the functions recommends placing the curly bracket { in its own line